gprofiler/utils/__init__.py

```diff
@@ -32,6 +32,12 @@
 from granulate_utils.linux.process import process_exe
 from psutil import Process
 
+from gprofiler.platform import is_linux, is_windows
+
+if is_windows():
+    import pythoncom
+    import wmi  # type: ignore
```

Comment:
type ignore can be moved to mypy.ini as well

Reply:
Done